native/src/parser.js

```diff
@@ -28,15 +28,14 @@ const GUESSING_ORDER = [
 ];
 
 export function guessParsing(code) {
+  let exceptions = [];
   for(let [fn, opts] of GUESSING_ORDER) {
     try {
       return fn(code, opts);
     } catch (ex) {
-      continue;
+      exceptions.push(ex)
     }
   }
+  throw new Error(exceptions.map((x) => x.message).join(","));
```

Comment:
Because `Error.prototype.message` is expected to be a `String`. I think if it is not, the browser can choose between displaying the empty string or displaying `ToString(message)`.